### ISSUE TITLE
Stuck at "[GameMemReader]: Connected to Among Us process (10649)"

### ISSUE THREAD

**Issue by ghost** [TauAkiou/amonguscapture-gtk#11]
Hey, I'm trying to get amonguscapture-gtk to work, but it seems to get stuck at

> [GameMemReader]: Connected to Among Us process (10649)

And never gets any further. "Current state" says "Disconnected". I have done the `cap_sys_ptrace` trick and even verified it with `getcap`.

I'm using Proton 5.13, and launching it through Steam with `PROTON_USE_WINED3D=1 %command%`.

Am I doing something wrong?

**Comment by TauAkiou** [comment]
i pushed an update to the 'current-version' branch. it should work, if you would like to help test it.

**Comment by ghost** [comment]
Thanks for taking the time!

I tried to compile it, but I got this message:

```
# dotnet build -c release
Microsoft (R) Build Engine version 16.10.0+4242f381a for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

/usr/share/dotnet/sdk/5.0.300/NuGet.targets(290,5): error MSB3202: The project file "/root/amonguscapture-gtk/AUOffsetManager/AUOffsetManager.csproj" was not found. [/root/amonguscapture-gtk/AmongUsCapture.sln]

Build FAILED.

/usr/share/dotnet/sdk/5.0.300/NuGet.targets(290,5): error MSB3202: The project file "/root/amonguscapture-gtk/AUOffsetManager/AUOffsetManager.csproj" was not found. [/root/amonguscapture-gtk/AmongUsCapture.sln]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.36
```

The `master` branch compiled just fine, so I don't think it's a problem with my environment. Is there a file missing in the commit, perhaps?

**Comment by TauAkiou** [comment]
AUOffsetManager wasn't added to the repo, it should be there now. However, I'm having some trouble getting the AUOffsetManager dll file to include itself, so you may have to build and copy it into the program directory manually.

**Comment by ghost** [comment]
Hey, sorry for the delay. The build worked after your update. I launched the application, but I still got the same result, stuck at Disconnected state.

However, it got a bit weird after that. After quitting the application, I tried to launch it again, but I was not able to run it successfully. It just got stuck for a while, and then printed this:

```
Found stale PID file containing 4108.
Unhandled exception. GLib.GException: Timeout was reached
   at GLib.Application.Register(Cancellable cancellable)
   at AmongUsCapture.Program.OpenGUI()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()

[1]+  Stopped                 ./AmongUsCapture
Aborted (core dumped)
```

I tried to remove the config directory, which got rid of the first message, but otherwise the results were the same, and now I cannot run it. Weird..

**Comment by TauAkiou** [comment]
This issue will be closed, as it is related to v1.0, which is no longer being worked on.
